src/AppInstallerCLITests/Completion.cpp

```diff
@@ -233,7 +233,7 @@ struct CompletionTestCommand : public Command
 
     using Command::Complete;
 
-    void Complete(Execution::Context& context, Execution::Args::Type valueType) const override
+    void Complete(CLI::Execution::Context& context, CLI::Execution::Args::Type valueType) const override

```

Comment:
Why did you need to do this?

Reply:
I believe that was needed when I was moving code from Core to Common (IIRC ChannelStream related code). It's not needed anymore, good catch!